### PR TITLE
Update system diagrams in the README

### PR DESCRIPTION
**Pull request by byliuyang**
### Description
The system diagrams in the README are for generic software systems. Readers have no clue how they are related to Short.

### Description
Replace generic diagrams with Short's specific design & implementation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -213,26 +213,29 @@ Short backend is built on top of
 [Uncle Bob's Clean Architecture](https://api.short-d.com/r/ca), the central
 objective of which is separation of concerns.
 
-![Clean Architecture](doc/eng/clean-architecture/clean-architecture.jpg)
-![Boundary](doc/eng/clean-architecture/boundary.jpg)
+![Clean Architecture](doc/clean-architecture/clean-architecture.jpg)
+
+![Short Backend](doc/clean-architecture/short-backend.jpg)
```
Comment (rohithbalaji123) [inline README.md:218]: In the image, is it should be Use Cases instead of User Cases.

File: README.md [hunk 2]
```diff
@@ -213,26 +213,29 @@ Short backend is built on top of
...
 
 Short adopts [Microservices Architecture](https://api.short-d.com/r/ms) to
 organize dependent services around business capabilities and to enable
 independent deployment of each service.
 
-![Microservice Architecture](doc/eng/microservices.jpg)
+![Short Cloud](doc/cloud/overall.jpg)
+[SSR](https://docs.google.com/document/d/1TuWexeKwhQh8JTytRAwST3XujBi0wTGExwJan-WfXWs), 
+[Toggle](https://docs.google.com/document/d/1TuWexeKwhQh8JTytRAwST3XujBi0wTGExwJan-WfXWs),
+[Status Page](https://docs.google.com/document/d/1pgRNnD8yAlEmj-sucS_FZ89LdvBy5zpKQ9OvILoBqDM), Search,
```
Comment (rohithbalaji123) [inline README.md:235]: Just wanted to inform that the "Search" exists in the line with no docs and whether it is intentional.

File: README.md [hunk 3]
```diff
@@ -68,15 +68,15 @@ from [source](https://short-d.com/r/ext-code)
...
-- Go v1.13.1
-- Node.js v12.12.0
-- Yarn v1.19.1
-- Postgresql v12.0 ( or use [ElephantSQL](https://short-d.com/r/sql) instead )
+- [Go](https://golang.org/doc/install) v1.13.1
+- [Node.js](https://nodejs.org/en/download/) v12.12.0
+- [Yarn](https://classic.yarnpkg.com/en/docs/install) v1.19.1
+- [PostgreSQL](doc/tutorial/POSTGRES.md) v12.0
 
 ### Create reCAPTCHA account
```
Comment (rohithbalaji123) [inline README.md:82]: Use `` tags instead of markdown's `[]()` to have better control over the size of the images.

Comment (rohithbalaji123) [reply README.md:82]: Preferably, we could move the individual setups to a new markdown file. That way, we focus our README with just the essentials and core features.
Moreover, for development, having one type of sign in is enough. We don't want to populate the README with detailed instructions to setup all kinds of logins. We can add links to a different .md file which can be referred for other login options. What do you think?

Comment (rohithbalaji123) [reply README.md:82]: Also, just a thought. Not sure how complicated it is. Is it possible to remove recaptcha validations in local dev environment?

File: README.md [hunk 4]
```diff
@@ -213,26 +213,29 @@ Short backend is built on top of
 [Uncle Bob's Clean Architecture](https://api.short-d.com/r/ca), the central
 objective of which is separation of concerns.
 
-![Clean Architecture](doc/eng/clean-architecture/clean-architecture.jpg)
-![Boundary](doc/eng/clean-architecture/boundary.jpg)
+![Clean Architecture](doc/clean-architecture/clean-architecture.jpg)
```
Comment (rohithbalaji123) [inline README.md:216]: Do we need this image here? The image by itself doesn't provide much information without the explanation from the blog. We have our infrastructure in the next image for devs to refer who already understood the architecture pattern.

File: README.md (hunk 2 above)
Comment (byliuyang) [reply README.md:235]: Yes, it is intended. No design doc for search yet.

File: README.md (hunk 1 above)
Comment (byliuyang) [reply README.md:218]: Good catch!

File: README.md [hunk 5]
```diff
@@ -339,7 +342,7 @@ func InjectGraphQlService(
 
 ### Database Modeling
 
-![Entity Relation Diagram](doc/eng/db/er-v1.jpg)
+![Entity Relation Diagram](doc/db/er.jpg)
 
 ### Feature Toggle
```
Comment (rohithbalaji123) [inline README.md:348]: The code example here is outdated.

Comment (rohithbalaji123) [reply README.md:348]: I guess, the developers can go through the project to understand the nuances of this feature. We can just mention such a feature exists in README.

File: README.md (hunk 1 above)
Comment (byliuyang) [reply README.md:218]: Fix 2 diagrams.

File: README.md (hunk 4 above)
Comment (byliuyang) [reply README.md:216]: @rohithbalaji123 People previously mentioned that they are interested in clean architecture when looking at the repo. So I kept this diagram here to attack developers.

File: README.md (hunk 3 above)
Comment (byliuyang) [reply README.md:82]: @rohithbalaji123 I agree everything you said. Should open issue to track this README clean up. For ReCaptcha, need to do some research.

File: README.md (hunk 4 above)
Comment (rohithbalaji123) [reply README.md:216]: Okay, agreed!

File: README.md (hunk 3 above)
Comment (byliuyang) [reply README.md:82]: https://github.com/short-d/short/issues/760

File: README.md (hunk 5 above)
Comment (byliuyang) [reply README.md:348]: https://github.com/short-d/short/issues/759

**Pull request merged**